src/annotator/frame-observer.js

```diff
@@ -124,8 +124,9 @@ export function onDocumentReady(frame) {
       frame.hasAttribute('src') &&
       frame.src !== 'about:blank'
     ) {
-      // Unfortunately, listening for 'DOMContentLoaded' on the iframeDocument
-      // doesn't work. Instead, we need to wait for a 'load' event to be triggered.
+      // Listening to `document.DOMContentLoaded` doesn't work because the
```

Comment:
```suggestion
      // Listening to `DOMContentLoaded` on the frame's document doesn't work because the
```

I guess you were trying to use "document.DOMContentLoaded" as a shorthand, but it is ambiguous as to what "document" refers to (eg. it _doesn't_ mean the `document` global).